### ISSUE TITLE
Bump submodules

### ISSUE THREAD

**Issue by JasonGross** [mit-plv/bedrock2#99]
Once https://github.com/mit-plv/riscv-coq/pull/22 is merged, please bump the submodules for coqutil and riscv-coq to the latest versions.  Once this is done, I can make bedrock2 use COQPATH for its dependencies, and then we can work on making bedrock2 itself a non-insane dependency.

**Comment by samuelgruetter** [comment]
I bumped the submodules, but then on trying to rebuild, I got

```
File "/home/sam/Documents/git/specbedrock/bedrock2/bedrock2/src/bedrock2/AbsintWordToZ.v", line 3, characters 0-66:
Error:
The file /home/sam/Documents/git/specbedrock/bedrock2/deps/coqutil/src/coqutil/Z/div_mod_to_equations.vo contains library
coqutil.Z.div_mod_to_equations and not library
coqutil.coqutil.Z.div_mod_to_equations
```

The offending line is

```
From coqutil Require Import Tactics.rdelta Z.div_mod_to_equations.
```

I don't understand, because I never use `From ... Import ...` myself, any idea?

**Comment by JasonGross** [comment]
You need to update all of the `-Q coqutil/src coqutil` to `-Q coqutil/src/coqutil coqutil `, because the base folder got moved

**Comment by samuelgruetter** [comment]
Oh I see thx I got confused and thought all of this was already done in https://github.com/mit-plv/bedrock2/pull/100

**Comment by JasonGross** [comment]
Thanks!
